### PR TITLE
Feat/how it works section

### LINKED ISSUE #7: Create How it Works Section
## 📄 Issue Description
Implement the How it Works section of the landing page based on the Figma design. This section showcases three stages: Register & Setup, Create and ShareLink, and Receive Payments.

## 🔍 Steps

1. Analyze the Figma design for the "How it Works" section.
2. See how the components of the current landing page are implemented and created.
3. Create a reusable component for each stage within the "How it Works" section.
4. Add appropriate icons or visuals for each stage.
5. Ensure the layout is responsive and visually consistent.

## ✅ Acceptance Criteria

- The "How it Works" section matches the Figma design.
- Each stage (Register & Setup, Create and ShareLink, Receive Payments) is clearly represented with corresponding visuals or icons.
- Section is fully responsive and accessible.
- Components must be separated in their individual folder per section. In this case, use the how-it-works folder in components.

## 🌎 References
[Figma Design](https://github.com/PayStell/paystell-landing/blob/main/landingPage.md)

### PR DESCRIPTION
**Pull request by kimcascante**
# Pull Request Overview

## 📝 Summary
This PR implements the "How it Works" section of the landing page as per the Figma design. It showcases three stages: Register & Setup, Create and ShareLink, and Receive Payments, using reusable and responsive components.

### 🔗 Related Issues
- Closes #7 

## 🔄 Changes Made
Analyzed Figma design for the "How it Works" section.
Using and updating reusable components for each stage.
Integrated corresponding icon and visuals for each stage.
Ensured full responsiveness.
Components are accessible and consistent with the overall landing page design.

## 🖼️ Current Output
Desktop View
Captura de pantalla 2024-12-16 a la(s) 10 37 22 p  m

Mobile View
Captura de pantalla 2024-12-16 a la(s) 10 38 21 p  m
Captura de pantalla 2024-12-16 a la(s) 10 38 34 p  m
Captura de pantalla 2024-12-16 a la(s) 10 38 41 p  m

## 🧪 Testing
Verified layout matches the Figma design
Tested responsiveness on devices with different screen sizes using Chrome DevTools.

## 💬 Comments
I’m missing the background image with the lines and circles as seen in the Figma design. I’d appreciate it if you could provide the asset or share any advice regarding its integration.

I’ll be more than happy to update anything as needed. Suggestions or feedback are always welcome! 😊

### CODE REVIEW

File: src/components/sections/HowItWorks.tsx [hunk 1]
```diff
@@ -1,55 +1,92 @@
-import { ArrowRight } from "lucide-react";
-
 const steps = [
```
Comment (MPSxDev) [inline src/components/sections/HowItWorks.tsx:1]:
```suggestion
import { Badge } from "../ui/badge";
import Image from "next/image";

const steps = [
```

File: src/components/sections/HowItWorks.tsx [hunk 2]
```diff
@@ -1,55 +1,92 @@
...
-		<section id="how-it-works" className="py-24 bg-background">
-			<div className="container">
-				<div className="text-center max-w-3xl mx-auto">
+		<section id="how-it-works" className="py-24 bg-background" aria-labelledby="how-it-works-title">
+			<div className="container w-full max-w-full">
+				<div className="flex justify-center items-center mb-4">
+					<div className="px-3 py-1 text-sm font-semibold rounded-full bg-[rgb(229,247,255)] text-[rgb(1,158,255)] flex items-center gap-1" >
+						BENEFITS <img src="/files/star.png" alt="start in benefits" className="w-5 h-5"/>
+					</div> 
+				</div>
```
Comment (MPSxDev) [inline src/components/sections/HowItWorks.tsx:64]:
```suggestion
				<div className="flex justify-center items-center mb-4">
            <Badge className="w-fit bg-primary/10 px-3 rounded-xl hover:bg-primary/10">
              <h2 className="text-lg font-bold text-primary mr-3">HOW IT WORKS</h2>
            <Image src="/star.svg" width={28} height={28} alt="star" />
          </Badge>
				</div>	
```

**Review by MPSxDev — CHANGES_REQUESTED** [review]
great job just those minor adjustments @kimcascante

File: src/components/sections/HowItWorks.tsx (hunk 2 above)
Comment (kimcascante) [reply src/components/sections/HowItWorks.tsx:64]: Hi @MPSxDev changes requested looks like this, could you please confirm? text should be HOW IT WORKS? and image position changed?

image

Comment (MPSxDev) [reply src/components/sections/HowItWorks.tsx:64]: The image is in the source repo, it will be ok if I merge it, thank you great job

Comment (MPSxDev) [reply src/components/sections/HowItWorks.tsx:64]: You can commit the changes

Comment (kimcascante) [reply src/components/sections/HowItWorks.tsx]: @MPSxDev I already committed the changes, let me know please if something else is needed

**Review by MPSxDev — APPROVED** [review]
great job thank you very much @kimcascante

**Pull request merged**